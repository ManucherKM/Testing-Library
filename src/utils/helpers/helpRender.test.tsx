import renderComponent from './helpRender'

describe('Тестирование helpRender', () => {
	//Чтобы обновить Snapshot нужно запустить тесты с флагом --updateSnapshot
	test.each([1, 2, 3, 4, 5, 'asd', 7, 8, 9, 10, 11, 12, 13, 14, 15])(
		'Правильное значение',
		a => {
			let jsx = <div>test {a}1</div>
			let res = renderComponent(jsx, `/${a}`)
			expect(res.baseElement.textContent).toMatchSnapshot()
		}
	)
})
