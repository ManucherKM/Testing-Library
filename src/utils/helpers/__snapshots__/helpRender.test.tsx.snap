// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Тестирование helpRender Правильное значение 1`] = `"test 11"`;

exports[`Тестирование helpRender Правильное значение 2`] = `"test 21"`;

exports[`Тестирование helpRender Правильное значение 3`] = `"test 31"`;

exports[`Тестирование helpRender Правильное значение 4`] = `"test 41"`;

exports[`Тестирование helpRender Правильное значение 5`] = `"test 51"`;

exports[`Тестирование helpRender Правильное значение 6`] = `"test asd1"`;

exports[`Тестирование helpRender Правильное значение 7`] = `"test 71"`;

exports[`Тестирование helpRender Правильное значение 8`] = `"test 81"`;

exports[`Тестирование helpRender Правильное значение 9`] = `"test 91"`;

exports[`Тестирование helpRender Правильное значение 10`] = `"test 101"`;

exports[`Тестирование helpRender Правильное значение 11`] = `"test 111"`;

exports[`Тестирование helpRender Правильное значение 12`] = `"test 121"`;

exports[`Тестирование helpRender Правильное значение 13`] = `"test 131"`;

exports[`Тестирование helpRender Правильное значение 14`] = `"test 141"`;

exports[`Тестирование helpRender Правильное значение 15`] = `"test 151"`;
